bundle/edu.gemini.model.p1.pdf/src/main/scala/edu/gemini/model/p1/pdf/P1PDF.scala

```diff
@@ -106,19 +108,29 @@ object P1PDF {
    * This method also merges the attached pdf file to the end of the resulting pdf.
    */
   def createFromFile (xmlFile: File, template: Template, pdfFile: File) {
-    Try(createFromNode(XML.loadFile(xmlFile), template, pdfFile, Option(xmlFile.getParentFile))).getOrElse(createFromNode(XML.loadString(Source.fromFile(xmlFile, "latin1").mkString), template, pdfFile, Option(xmlFile.getParentFile)))
+    Try(createFromNode(XML.loadFile(xmlFile), template, pdfFile, Option(xmlFile.getParentFile)))
+      .getOrElse {
+        val source = Source.fromFile(xmlFile, "latin1")
+        createFromNode(XML.loadString(source.mkString), template, pdfFile, Option(xmlFile.getParentFile))
+        source.close()
+      }
   }
 
   /**
    * Creates a pdf from a given xml element and template and writes the resulting pdf file to the output folder.
    * This method also merges the attached pdf file to the end of the resulting pdf.
    */
   def createFromNode (xml: Node, template: Template, pdfFile: File, workingDir:Option[File] = None) {
+    // REL-3772: Do not display PI information for FT proposals.
+    val workingTemplate = {
+      val isFT: Boolean = (xml \ "proposalClass" \\ "fastTurnaround").theSeq.nonEmpty
+      if (isFT) template.copy(investigatorsList = InvestigatorsListOption.NoList) else template
```

Comment:
Hmm, I thought we'd update the xslt and remove the page altogether, though this seems to be a valid alternative. Does the whole page disappear or is it just empty? Does that make any difference?

Reply:
We only want it for fast turnaround, and I figured since there was a `Template` class with a specific option for investigator information, it was probably the best way to go since the code was already in place to omit them.

I'm trying to upload an example but github is complaining about not being able to process the file.

Reply:
It allowed me to add the files to the description up above in case you want to take a look.

Reply:
(The last page is blank because I just use a blank PDF for the PDF attachment.)

Reply:
Also, another benefit: we don't have to edit multiple XSLT files, which we'd have to do if we went that route. (Wouldn't we have to change the one for each format?)